blip-tap.md

```diff
@@ -919,149 +919,251 @@ order to clamp down on price volatility exposure.
 TODO(roasbeef): explain AMP/keyspend, basically unsolicited quote req, can
 accept if favorable
 
-
 #### RFQ Negotiation
 
+This section describes the Request-For-Quote (RFQ) negotiation process. A
+successful outcome of the RFQ process between two peers is an agreed currency
+exchange rate. This rate can be used to convert a TAP asset amount into BTC, or
```

Comment:
nit: shouldn't use "TAP", should just say "Taproot Asset amount".

Reply:
TAP is used 99 times in that file, all over the doc. To be consistent, I will use it in the changes in this PR also. And we can make another PR to change it throughout later. Does that sound reasonable to you?

Reply:
Ah, I didn't notice... Yeah, sounds good to me.